README.md

```diff
@@ -232,6 +232,24 @@ The environment is accessible through the following URLs:
 >
 > Links to share a collection or a dataset via LinkedIn, X or Facebook will now open in a new tab instead of a popup.
 
+</details>
+
+<details>
+  <summary><strong>Requirements for deploying the SPA</strong></summary>
+
+> ### Deploying a Keycloak Instance for Built-in Users Authentication  
+>  
+> To enable authentication for Dataverse built-in user accounts in the SPA, you must deploy a Keycloak instance as an OIDC authentication provider and broker for the SPA.  
+>  
+> However, using Keycloak is not mandatory. If you do not require authentication for Dataverse built-in users, you can use any OIDC provider to handle user authentication in the SPA.  
+>  
+> For supporting built-in users, Keycloak must be properly configured and integrated with the Dataverse backend using the [Dataverse Built-in Users SPI][dv_repo_builtin_users_spi_provider].  
+>  
+> On the SPA side, ensure that the PKCE environment variables are set up to connect to the chosen OIDC provider for authentication.  
+>  
+> Additionally, to allow the SPI to authenticate users against the Dataverse database, the Dataverse database must be accessible from the Keycloak service within the deployed infrastructure.  
```

Comment:
@donsizemore can we please try this on https://beta.dataverse.org ? Make the database accessible to https://beta-keycloak.dataverse.org ?

Reply:
absolutely. I was waiting on somebody from IQSS to tell me "yes, make this change for beta"